doc/Language/traps.pod6

```diff
@@ -1990,5 +1990,22 @@ my @a = <1 2 3>;
 say @a.grep( 2 ~~ *.Int ); # OUTPUT: «(2)␤»
 =end code
 
+=head1 Closures
+
+Raku supports closures in many places. Not all of them are supposed to work.
+
```

Comment:
Hum, what do you mean exactly here? If it "supports" them, shouldn't it be supposed to work?

Reply:
Please take a look at this thing.